File: workspace-servidor/flotix2021/src/main/java/com/flotix/response/bean/ServerResponseVehiculo.java
package com.flotix.response.bean;

import java.util.List;

import com.flotix.dto.VehiculoDTO;

public class ServerResponseVehiculo {

	private String idVehiculo = null;

	private List<VehiculoDTO> listaVehiculo = null;

	private ErrorBean error = new ErrorBean();

	public ServerResponseVehiculo() {
		super();
	}

	public String getIdVehiculo() {
		return idVehiculo;
	}

	public void setIdVehiculo(String idVehiculo) {
		this.idVehiculo = idVehiculo;
	}

	public List<VehiculoDTO> getListaVehiculo() {
		return listaVehiculo;
	}

	public void setListaVehiculo(List<VehiculoDTO> listaVehiculo) {
		this.listaVehiculo = listaVehiculo;
	}

	public ErrorBean getError() {
		return error;
	}

	public void setError(ErrorBean error) {
		this.error = error;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelResponse/ServerResponseVehiculo.cs
using Flotix2021.Helpers;
using Flotix2021.ModelDTO;
using System.Collections.Generic;

namespace Flotix2021.ModelResponse
{
    public class ServerResponseVehiculo
    {
        private string _idVehiculo;

        public string idVehiculo
        {
            get { return _idVehiculo; }
            set { _idVehiculo = value; }
        }

        private List<VehiculoDTO> _listaVehiculo;

        public List<VehiculoDTO> listaVehiculo
        {
            get { return _listaVehiculo; }
            set { _listaVehiculo = value; }
        }

        private ErrorBean _error;

        public ErrorBean error
        {
            get { return _error; }
            set { _error = value; }
        }

    }
}
